src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -460,10 +460,12 @@ public void onClose(int i, String s) {
                     toggleStdout.disable();
                     OutputStream stdin = watch.getInput();
                     PrintStream in = new PrintStream(stdin, true, StandardCharsets.UTF_8.name());
+                    boolean windows = sh.equals("cmd");
```

Comment:
IIUC from https://github.com/jenkinsci/kubernetes-plugin/blob/18d73fb7efd6d7702f53cd7d2f17b5d28c414f9c/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java#L371

Seems a bit fragile, since `shell` could in principle be overridden on Windows, but OK. Maybe clearer to introduce a constant here.